assignments/assignment02.rb

```diff
@@ -73,3 +109,6 @@ def pluck(ary)
 # - daily balance
 # - summary:
 #   - starting balance, total deposits, total withdrawals, ending balance
+
+
+### ran into ruby version problems, will have to update with problem 4 later
```

Comment:
OK. Please work on completing this, you'll learn a lot by actually working through this problem.